### PR TITLE
Tickets/dm 26846

### PR DESCRIPTION
**Pull request by jgates108**

### CODE REVIEW

File: replicator_setup/replication.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ 
+INSERT INTO `config_worker` VALUES ('worker', 1, 0, 'localhost', NULL, 'localhost', NULL, NULL, 'localhost', NULL, NULL, '192.168.1.143', NULL, NULL, '192.168.1.143', NULL, NULL);
+ 
+-- This is the default database family for LSST production
+ 
+INSERT INTO `config_database_family` VALUES ('production', 1, 340, 3, 0.01667);
+ 
+SET SQL_MODE=@OLD_SQL_MODE ;
+SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS ;
+SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS ;
```
Comment (iagaponenko) [inline replicator_setup/replication.sql:70]: Wby do you want to set up the Replication system for the data generator?

File: bin/run_distribserver.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# after deleteing localConfig.
+rm -rf localConfig
+cp -r configs/$1 localConfig
+
+# Do any specific building for the database
+/bin/bash localConfig/prepare.sh
+ls -la localConfig
+
+# start the server
+python bin/datagenserver.py ${@: 2}
```
Comment (iagaponenko) [inline bin/run_distribserver.sh]: No newline in the end of the file.

File: container/kubernetes/dgenclient.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      labels:
+        app: datagenclient
+    spec:
+      hostNetwork: True
+      containers:
+      - name: datagen
+        image: qserv/dax_data_generator:master
+        imagePullPolicy: Always
+        # "-H192.168.1.143" needs to be set to the correct computer
+        command: ["/home/qserv/dax_data_generator/scripts/run_datagenclient.sh", "-H192.168.1.143", "-r"]
```
Comment (iagaponenko) [inline container/kubernetes/dgenclient.yaml]: Why not put arguments into:
```
args: ["-H192.168.1.143", "-r"]
```

File: replicator_setup/replication.sql (hunk 1 above)
Comment (jgates108) [reply replicator_setup/replication.sql:70]: That is only for local testing on a single computer.

**Review by fritzm — APPROVED** [review]

**Pull request merged**